.github/workflows/greetings.yml

```
@@ -13,4 +13,15 @@ jobs:
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         issue-message: "Greetings new comers! Hope we can help you with your questions! 新人你好！希望我们能帮助你解答你的问题！"
-        pr-message: "Greetings new comers! Thanks for helping us become better! 新人你好！感谢你能帮我们变得更好！"
+        pr-message: |
+          Greetings new comers! Thanks for helping us become better! 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)